pkg/config/config.go

```diff
@@ -81,10 +81,6 @@ type Config struct {
 	SavedCredentials    map[string]*LoginCredential `json:"saved_credentials,omitempty"`
 	LocalPorts          *LocalPorts                 `json:"local_ports,omitempty"`
 
-	// Deprecated
-	AnonymousId string `json:"anonymous_id,omitempty"`
-	Ver         string `json:"version,omitempty"`
```

Comment:
These fields can't be deleted. Users who still have them in their configuration file will get a serialization error upon running any command.

Reply:
Oh okay gotcha. Do we want these fields to be in the whitelist? (I'm assuming no since they're deprecated).

Reply:
Yep, let's leave them out!